sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/strategies/MessageListenerStrategy.java

```diff
@@ -45,40 +44,21 @@ public abstract class MessageListenerStrategy<K, V> {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MessageListenerStrategy.class);
 
-  protected ConsumerTopicMessageHistogram consumerProcessedMsgHistogram;
-
   private Map<TopicPartition, OffsetAndMetadata> offsetsToCommitOnClose = new HashMap<>();
   private Set<String> metadataFields = Set.of();
 
   /**
-   * @param consumerTopicMessageHistogram the histogram that tracks time for consuming single
-   *     records
-   * @deprecated in favour of {@link #init(ConsumerTopicMessageHistogram, Set)}
-   */
-  @Deprecated
-  @SuppressWarnings({"DeprecatedIsStillUsed", "java:S6355", "java:S1133"})
-  public void init(ConsumerTopicMessageHistogram consumerTopicMessageHistogram) {
-    init(consumerTopicMessageHistogram, MetadataContext.metadataFields());
-  }
-
-  /**
-   * @param consumerTopicMessageHistogram the histogram that tracks time for consuming single
-   *     records
    * @param metadataFields the configured {@link MetadataContext#metadataFields()}
    */
-  public void init(
-      ConsumerTopicMessageHistogram consumerTopicMessageHistogram, Set<String> metadataFields) {
-    this.consumerProcessedMsgHistogram = consumerTopicMessageHistogram;
+  public void init(Set<String> metadataFields) {
```

Comment:
Since this is a public API, could you also add this to the migration guide?

Reply:
Added kafka section to migration guide